lib/OnyxUtils.ts

```diff
@@ -752,7 +756,8 @@ function sendDataToConnection<TKey extends OnyxKey>(mapping: Mapping<TKey>, val:
         return;
     }
 
-    (mapping as DefaultConnectOptions<TKey>).callback?.(val, matchedKey as TKey);
+    const valWithoutNestedNulls = utils.removeNestedNullValues(val);
```

Comment:
NAB, we could use valuesWithoutNestedNulls

Reply:
updated!